packages/fxa-auth-server/test/remote/subscription_tests.js

```diff
@@ -179,13 +179,14 @@ describe('remote subscriptions:', function() {
 
     describe('with a subscription', () => {
       const subscriptionId = 'sub_12345';
+      const date = Date.now() * 1000;
       beforeEach(() => {
         mockStripeHelper.customer = async (uid, email) => ({
           subscriptions: {
             data: [
               {
                 id: subscriptionId,
-                created: Date.now() / 1000,
+                created: date / 1000,
```

Comment:
Did you mean `date`?

Reply:
yep

Reply:
Updated

Reply:
Ah, ok. I wasn't aware of the distinction between the `created` and `createdAt` values